### ISSUE TITLE
Is it possible to add data values on tr ?

### ISSUE THREAD

**Issue by VCarrasco** [aslagle/reactive-table#263]
I need to add to each "tr" current values in data elements to order the table automatically depending different values like:

<tr data-player-points='100' data-player-room=1>
....
<tr data-player-points='90' data-player-room=2>
....
etc

is it possible now?

Cheers.

**Comment by VCarrasco** [comment]
I guess it can be fixed doing the next:

{ label: 'field_1', key: 'field_1', sortByValue: true, sortOrder: 0, sortDirection: 'ascending'
...
{ label: 'field_2', key: 'field_2', sortByValue: true, sortOrder: 0, sortDirection: 'ascending'
...



Could you let me know if it is right?



Thank you.
P.s. Excellent package.

**Comment by aslagle** [comment]
I'm not sure exactly what you're trying to do, but you might want to have sortOrder: 0 for one field and sortOrder: 1 for the other, to control which one is the primary sort and which one is secondary.

**Comment by VCarrasco** [comment]
You are right about sortOrder (was a mistake writing)  I'm using just one column to sort it at the moment (all the logic is there).

But if I want to sort by some columns using 'cellString' ? 

I hope next example helps.

{ label: 'room', key: 'room', sortByValue: true, sortOrder: 0, sortDirection: 'ascending'
Player 1... 'span cellString="2" ...Room 2... /span'
Player 2... 'span cellString="1" ...Room 1... /span'
Player 3... 'span cellString="1" ...Room 1... /span'
Player 4... 'span cellString="2" ...Room 2... /span'

{ label: 'points', key: 'points', sortByValue: true, sortOrder: 1, sortDirection: 'ascending'
Player 1... 'span cellString="15" ...15 points... /span'
Player 2... 'span cellString="30" ...30 points... /span'
Player 3... 'span cellString="20" ...20 points... /span'
Player 4... 'span cellString="10" ...10 points... /span'


The result ordering ASC  should be

Player 3 ... Room 1 ... 20 points 
Player 2 ... Room 1 ... 30 points
Player 4 ... Room 2 ... 10 points
Player 1 ... Room 2 ... 15 points  


Should I use  'sortByValue: true, sortOrder: x, sortDirection: ascending' in both fields to get it ?


Thank you.

**Comment by aslagle** [comment]
That looks right to me, if I understand correctly. Is it working? If you use sortByValue, it won't be using the cellString, it will use the actual value for that cell - but if the value is what you're putting in cellString then it's probably better to use the numerical value.
